pkg/config/engine/crio/crio.go

```diff
@@ -65,11 +80,13 @@ func (c *Config) AddRuntime(name string, path string, setAsDefault bool) error {
 
 	config := *c.Tree
 
-	// By default we extract the runtime options from the runc settings; if this does not exist we get the options from the default runtime specified in the config.
-	runtimeNamesForConfig := []string{"runc"}
+	// By default, we extract the runtime options from the settings of the runtime marked as the "default_runtime";
+	// if this does not exist we get the options from the "runc" runtime.
```

Comment:
Should we also include `crun` in this list for `crio`?

Reply:
Yes, I am open to that

Reply:
Done